astropy/utils/decorators.py

```diff
@@ -936,3 +936,93 @@ def set_docstring(obj):
             obj.__doc__ = doc.format(*args, **kwargs)
         return obj
     return set_docstring
+
+
+def replaced_parameter(oldkeyword, newkeyword, since):
+    """Enable a function with a new keyword signature, to still use the
+    deprecated old keyword.
+
+    The decorated function should follow the new signature: the
+    definition should use ``newkeyword`` instead of ``oldkeyword``.
+    The new parameter should behave the same as the old parameter.
+
+    An AstropyDeprecationWarning is given if ``oldkeyword`` is used,
+    and a TypeError is raised if both ``oldkeyword`` and
+    ``newkeyword`` are used in the decorated function.
+
+    Parameters
+    ----------
+
+    oldkeyword : str
+        The keyword in the function signature that has been replaced
+
+    newkeyword : str
+        The keyword in the function signature replaces oldkeyword
+
+    since : str
+        The release when this change was made.
+
+
+    Raises
+    ------
+
+    TypeError
+        If both ``newkeyword`` and ``oldkeyword`` are used.
+
+    Example
+    -------
+
+    This decorator originated when replacing the `clobber` parameter
```

Comment:
Double backticks. Otherwise, Sphinx will try to link them somewhere and report them as broken links.